### PR TITLE
Add party guide to resource

### PR DESCRIPTION
**Pull request by hpdang**

### CODE REVIEW

File: _docs/resources/how-to-organize-release-party.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+title: How to organize a release party
+author: OST
+date: 2018-09-25
+index: 1
```
Comment (perploug) [inline _docs/resources/how-to-organize-release-party.md:5]: make sure the navigation index is correct

File: _docs/resources/how-to-organize-release-party.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+* Speaker
+* Receptionist (take care of check-in) 
+* Caterer (take car of food & drink) 
+
+>*Make sure you keep documentation of the planning progress and share it with all the organizing team members. Remember to plan to back-up in case you or your colleague for any reason might not be able to run the event.*
+
+### Step 5: Open up registration
+Registration gives you an idea how many people might turn up and the profile of audience which could be useful for you to plan your communication strategy more effectively at the event. 
+
+For registration, you could use Google Form, Google Calendar, or open source solution [Eventyay](https://eventyay.com/).
```
Comment (perploug) [inline _docs/resources/how-to-organize-release-party.md:59]: Have you checked if Eventyay is something Zalandos can use - if its a service - then we need consider GDPR

File: _docs/resources/how-to-organize-release-party.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+For registration, you could use Google Form, Google Calendar, or open source solution [Eventyay](https://eventyay.com/). 
+
+Send out personal invitations to people who have unique background that could be valued at the event. People often appreciate to receive personal attention. 
+
+>*Free events, no matter where in the world they take place, receive around 50% turnout is a good scenario. This means if you want 50 people at your event, you need to register 100 tickets. And don’t forget to record who actually show up at your event.*
+
+### Step 6: Announce your event publicly
+A few things that you can do: 
+
+* Create a facebook event including your sign-up form.
```
Comment (perploug) [inline _docs/resources/how-to-organize-release-party.md:68]: should we include meetup.com in this list ?

File: _docs/resources/how-to-organize-release-party.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+20 mins Presentation No.2 
+
+Ends + Time to mingle
+ 
+> *You could consider to invite a special guest (a famous speaker) to attract a wider group of audience.*
+
+### Step 2: Select a date & time 
+It is a good idea to check out company calendar or local events calendar (meetup.com) to avoid a clash with any event on a similar topic. Schedule at the end of the workday, but also keep in mind that community members have family to get home to. We would recommend set the date of your event to be at least 3 weeks in the future. 
+
+### Step 3: Lockdown venue, logistics, budget
```
Comment (perploug) [inline _docs/resources/how-to-organize-release-party.md:40]: I think lockdown refers to security - Should probably be "Secure venue, logistics and budget"

**Review by perploug — CHANGES_REQUESTED** [review]

File: _docs/resources/how-to-organize-release-party.md (hunk 2 above)
Comment (hpdang) [reply _docs/resources/how-to-organize-release-party.md:59]: @perploug it is an open source solution, people can set it up on their own server

**Comment by hpdang** [comment]
@perploug I created a new PR including this article, can we close this?

File: _docs/resources/how-to-organize-release-party.md (hunk 3 above)
Comment (hpdang) [reply _docs/resources/how-to-organize-release-party.md:68]: only meetup organizer can publish a meetup. Organizers pay a fee. I already included this information on the promotion channels article

**Pull request closed without merging**